src/arch/Sound/RageSoundDriver_WaveOut.cpp

```diff
@@ -103,38 +113,55 @@ int64_t RageSoundDriver_WaveOut::GetPosition() const
 {
 	MMTIME tm;
 	tm.wType = TIME_SAMPLES;
-	MMRESULT ret = waveOutGetPosition( m_hWaveOut, &tm, sizeof(tm) );
+	MMRESULT ret = waveOutGetPosition( waveout_handle_, &tm, sizeof(tm) );
   	if( ret != MMSYSERR_NOERROR )
 		FAIL_M( wo_ssprintf(ret, "waveOutGetPosition failed") );
 
 	return tm.u.sample;
 }
 
 RageSoundDriver_WaveOut::RageSoundDriver_WaveOut()
+	: waveout_handle_(nullptr)
+	, soundevent_handle_(CreateEvent(nullptr, false, true, nullptr))
+	, wo_buffers_{}
+	, wo_samplerate_(0)
+	, wo_shutdown_(false)
+	, wo_last_cursor_position_(0)
+	, wo_init_success_(false)
+	, wo_frames_per_block_(0)
+	, wo_num_blocks_(1)
+	, wo_blocksize_(0)
+	, MixingThread()

```

Comment:
Seems like MixingThread is first not last. I suppose might as well rename that variable as well while we're here.